src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -34,13 +34,12 @@
 public class TranslogStats implements ToXContent, Streamable {
 
     private long translogSizeInBytes = 0;
-    private int estimatedNumberOfOperations = 0;
+    private int estimatedNumberOfOperations = -1;
```

Comment:
Why not leave this at 0? You can keep the assert and then you don't have to worry about someone serializing an empty `TranslogStats` and running into problems with `out.writeVInt(estimatedNumberOfOperations)`